snuba/migrations/operations.py

```diff
@@ -48,3 +51,34 @@ def __init__(self, storage_set: StorageSetKey, table_name: str) -> None:
 
     def format_sql(self) -> str:
         return f"DROP TABLE IF EXISTS {self.table_name};"
+
+
+class TableEngine:
+    pass
+
+
+class ReplacingMergeTree(TableEngine):
+    def __init__(self, version_column: str):
+        self.__version_column = version_column
+
+    def __str__(self) -> str:
+        return f"ReplacingMergeTree({self.__version_column})"
+
+
+class CreateTable(SqlOperation):
+    def __init__(
+        self,
+        storage_set: StorageSetKey,
+        table_name: str,
+        columns: Sequence[Column],
```

Comment:
Any reason not to use the ColumnSet here ?

Reply:
I wanted to avoid relying on higher level classes from the query schema. I plan to remove the `for_schema` method from here later so there won't be any duplication.